### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by khanhct**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# test-cicd
\ No newline at end of file
+# test-cicd
+# Intro
```
Comment (khanhct) [inline README.md:2]: Great stuff!

Comment (khanhct) [inline README.md:2]: Great stuff!

Comment (khanhct) [inline README.md:2]: Great stuff!

Comment (khanhct) [inline README.md:2]: Great stuff!

Comment (khanhct) [reply README.md:2]: # test

Comment (khanhct) [inline README.md:2]: dfdsfdsfdsfdsf

Comment (khanhct) [inline README.md:2]: dfdsfdsfdsfdsf

Comment (khanhct) [inline README.md:2]: dfdsfdsfdsfdsf

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
-# test-cicd
\ No newline at end of file
+# test-cicd
```
Comment (khanhct) [inline README.md:1]: dfdsfdsfdsfdsf

Comment (khanhct) [inline README.md:1]: dfdsfdsfdsfdsf

File: README.md (hunk 1 above)
Comment (khanhct) [inline README.md:2]: dfdsfdsfdsfdsf

Comment (khanhct) [inline README.md:2]: dfdsfdsfdsfdsf

Comment (khanhct) [inline README.md:2]: dfdsfdsfdsfdsf

Comment (khanhct) [inline README.md:2]: dfdsfdsfdsfdsf

Comment (khanhct) [inline README.md:2]: dfdsfdsfdsfdsf

Comment (khanhct) [inline README.md:2]: dfdsfdsfdsfdsf

Comment (khanhct) [inline README.md:2]: dfdsfdsfdsfdsf

Comment (khanhct) [inline README.md:2]: 1111111111111111111

Comment (khanhct) [inline README.md:2]: 1111111111111111111

Comment (khanhct) [inline README.md:2]: 1111111111111111111

Comment (khanhct) [inline README.md:2]: The addition of an 'Intro' section to the README.md is a good practice for introducing the project.

File: test.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
+const parseLastDiff = (gitDiff) => {
+  gitDiff = gitDiff.replace(/\n\\ No newline at end of file/, '')
```
Comment (khanhct) [inline test.py:2]: The code needs refinement in terms of naming conventions, error handling, and potential performance issues.

File: README.md (hunk 1 above)
Comment (khanhct) [inline README.md:2]: The addition of an 'Intro' section to the README.md is a good practice for introducing the project.
Reason:

File: test.py (hunk 3 above)
Comment (khanhct) [inline test.py:2]: The code needs refinement in terms of naming conventions, error handling, and potential performance issues.
Reason: 1. **Naming Conventions**: The function name `parseLastDiff` doesn't follow Python naming conventions which should use snake_case for function names. Also, the usage of `const` suggests JavaScript or TypeScript syntax which is incorrect for Python.

2. **Unnecessary Operations**: The line `gitDiff = gitDiff.replace(/
\ No newline at end of file/, '')` seems to remove a specific string but the regex syntax is for JavaScript, not Python. In Python, it should be `gitDiff = gitDiff.replace(r'
\ No newline at end of file', '')`.

3. **Error Handling**: The function attempts to perform operations that might fail (like `split(',')` or regular expressions) without adequate error handling or checks.

4. **Potential Performance Issue**: The use of `reverse()` and then `find()` might not be the most efficient for large diffs.

5. **Syntax Issues**: The syntax `return parseLastDiff($input.item.json.diff)` at the end implies use of a variable `$input` which is not defined in the provided code snippet, and again suggests JavaScript/TypeScript which conflicts with the file extension `.py`.

6. **Type Hinting and Docstrings**: There are no type hints or docstrings which would enhance readability and maintainability in Python.

Comment (khanhct) [inline test.py:2]: The new code introduces a function `parseLastDiff` that effectively parses git diff information. It handles edge cases and provides structured output. The code is well-structured with clear logic. However, consider adding more comments for better readability and maintainability if this becomes part of a larger codebase.

Reason:

File: README.md (hunk 1 above)
Comment (khanhct) [inline README.md:2]: The change adds an introduction section which improves the document structure.

Reason:

Comment (khanhct) [inline README.md:2]: The change adds a new section header which improves the structure of the README.md file.

Reason:

File: test.py (hunk 3 above)
Comment (khanhct) [inline test.py:2]: The function `parseLastDiff` is well-structured but has some issues that need to be addressed.

Reason: 
1. **Syntax Error**: The function is defined with `const`, but JavaScript in a typical environment like Node.js or a browser expects `function` or `export` for module definitions if not in strict module mode. If this is intended for a specific environment or build system, it should be clarified.

2. **Unnecessary Complexity**: The handling of `lastLineFirstChar` to determine `lastOldLine` and `lastNewLine` could be simplified. The current logic might lead to confusion as it uses `null` when perhaps `undefined` or a specific value might be clearer.

3. **String Parsing**: Using regular expressions to parse the diff format could be brittle if the format changes slightly. Consider using a more robust diff parsing library if available.

4. **Function Usage**: The last line `return parseLastDiff($input.item.json.diff)` seems out of place within the function definition. It should be outside the function or clarified if this is part of a larger context (like a module export or a script execution).

File: README.md (hunk 1 above)
Comment (khanhct) [inline README.md:2]: The addition of an introductory section improves the readability and structure of the README.md file.

Reason:
